src/cases/containers/case-filter/case-filter.component.ts

```diff
@@ -36,7 +36,7 @@ export class CaseFilterComponent implements OnInit {
 
   ngOnInit(): void {
     this.fromCasesFeature = fromCasesFeature;
-    this.startButtonText = 'Start'; // TODO add this to some config file.
+    this.startButtonText = 'Start now'; // TODO add this to some config file.
```

Comment:
Is this still not going into a config file then?

Reply:
As discussed only text change is mentioned in the ticket

Reply:
Can you please move this string into the app.constants file @RiteshHMCTS ?

Reply:
I have moved the text to app.constants file